### DISCUSSION TITLE
JSON to Plain Text

### DISCUSSION THREAD

**Discussion by philmetzger** [ueberdosis/tiptap#3114] [General Questions] [answered]
Are there any utilities that I have not been able to find to convert JSON to plain text?
e.g. given `{"type":"doc","content":[{"type":"paragraph","content":[{"type":"text","text":"asdasdasdd"}]}]}` how could I get the plain text?

**Comment by sipec** [comment, marked as answer]
Yes, there's `generateText(json, extensions)`. It's not well documented, but https://github.com/ueberdosis/tiptap/pull/1875 explains how to use it.
